### ISSUE TITLE
Improve cache clearing with multilingual plugins

### ISSUE THREAD

**Issue by webtrainingwheels** [wp-media/wp-rocket#910]
As an example, with WPML, if you use the Clear Cache link to clear a specific language, the minified/combined files will also be deleted. This causes 404s on other language pages because the cache for those pages has not been deleted. 
Sample ticket: https://secure.helpscout.net/conversation/546305981/63173?folderId=377611

[On Slack](https://wp-media.slack.com/archives/C08N8J6VC/p1522095006000569), @arunbasillal suggested the following possibility: 
"A hot fix would be include this block: 
https://github.com/wp-media/wp-rocket/blob/master/inc/common/purge.php#L390-L401
Inside a `if( $lang == '' ) {`
Basically clean the files only when all languages are selected."
